src/lib/types.rs

```diff
@@ -1091,15 +1170,13 @@ impl Task {
                         self.run_task = Some(run_task);
                     }
 
-                    if self.dependencies.is_some() {
-                        let dependencies = self.dependencies.clone().unwrap();
-                        let mut modified_dependencies = vec![];
-
-                        for task in &dependencies {
-                            modified_dependencies.push(get_namespaced_task_name(namespace, &task));
-                        }
-
-                        self.dependencies = Some(modified_dependencies);
+                    if let Some(dependencies) = &self.dependencies {
+                        self.dependencies = Some(
+                            dependencies
+                                .iter()
+                                .map(|d| d.to_owned().with_namespace(namespace))
```

Comment:
d :)

Reply:
I usually only use them for lambdas and patterns but I agree they're more useful when you have rust_analyzer holding your hand! I'll get those fixed :)